controlFlow/taskTwo/main.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"strconv"
+	"strings"
+)
+
+func main() {
+	input := "1 9 3 4 -5"
+	var result string
+	var min, max int32
+
+	dividedInput := strings.Split(input, " ")
+
+	for i, v := range dividedInput {
+		if i == 0 {
+			intOfValue, err := strconv.Atoi(v)
+			if err != nil {
+				log.Fatal(err)
+			}
```

Comment:
This block is done for each element, so can be put outside of the `if`, to avoid duplications.

Reply:
Done.